include/dlaf/communication/kernels/p2p.h

```diff
@@ -70,15 +70,15 @@ template <class CommSender, class Sender>
   using dlaf::internal::whenAllLift;
   using dlaf::internal::withTemporaryTile;
 
-  auto recv = [dest, tag, pcomm = std::forward<CommSender>(pcomm)](auto const& tile_comm) mutable {
+  auto send = [dest, tag, pcomm = std::forward<CommSender>(pcomm)](auto const& tile_comm) mutable {
```

Comment:
For your information, I've applied this change here as well: https://github.com/eth-cscs/DLA-Future/pull/674/commits/e879a337bad9b51fa7494e6a5c765f94d32bc097 (since this has moved to a source file the change would possibly be missed otherwise...).